WordPress/Classes/Networking/AccountSettingsRemote.swift

```diff
@@ -56,9 +55,9 @@ class AccountSettingsRemote: ServiceRemoteREST {
                     }
             })
             return AnonymousDisposable() {
-                if let operation = operation {
-                    if !operation.finished {
-                        operation.cancel()
+                if let progress = progress {
+                    if !(progress.fractionCompleted == 1.0) {
+                        progress.cancel()
```

Comment:
I think I'd just call `progress?.cancel()` here. Is there a reason why we need to check for completion before calling cancel?